src/actions/signupSigninActions.js

```diff
@@ -1,10 +1,9 @@
 import { notify } from 'react-notify-toast';
 import axios from 'axios';
-import config from '../../dist/config';
 
 export const signUpUser = (user, history) => () => {
   axios
-    .post(`${config.baseURL}/register/`, user)
+    .post(`${process.env.baseURL}users/register/`, user)
```

Comment:
Add a trailing `/` before `users/register`. Fix in other places as well

Reply:
Thanks for that feedback @adesege,
I have implemented it